### ISSUE TITLE
[Question] PS4 SRD File UNPACK chunk images

### ISSUE THREAD

**Issue by l0ve2s** [jpmac26/DRV3-Sharp#5]
Do you have any plans to support PS4?
PS4 SRD.file UNPACK chunk images
please confirm
Thank you.

PC SRD UNPACK
![99](https://user-images.githubusercontent.com/88938174/203648035-9f6d6d91-d4a5-48cb-af1e-87f0f1f8e288.png)

PS4 SRD UNPACK
![999](https://user-images.githubusercontent.com/88938174/203648074-9666308d-3a35-4f7d-a8ca-be4774debe30.png)

Raw texture
![7](https://user-images.githubusercontent.com/88938174/203648107-33bb8f80-d299-4bc0-bf5a-62561439eb16.png)
[PC.startup_JP.zip](https://github.com/CaptainSwag101/DRV3-Sharp/files/10079235/PC.startup_JP.zip)
[PS4.startup_US.zip](https://github.com/CaptainSwag101/DRV3-Sharp/files/10079236/PS4.startup_US.zip)

**Comment by CaptainSwag101** [comment]
Hello, in some personal tests I did a few years ago I managed to get PS4 images working partially, but it remains rather buggy and incomplete at this time. I would like to add complete support but I do not know if I will be able to accomplish this. There are many programming challenges that must be overcome, and DRV3-Sharp still has several issues and limitations when working with PC files that I want to fix and improve as well.

To summarize: it might be possible to add support for PS4, and I would like to do it, but since I have limited time to work on these tools due to school and my job, I cannot guarantee that it will happen, since my primary focus is on PC.

**Comment by l0ve2s** [comment]
Thank you for your answer.
